pkg/kudoctl/cmd/install/install.go

```diff
@@ -132,8 +132,11 @@ func installCrds(crds *bundle.PackageCRDs, kc *kudo.Client, options *Options, se
 		return err
 	}
 
-	// Operator part
+	if err := kc.ValidateServerForOperator(crds.Operator); err != nil {
```

Comment:
the full operator is passed... I'm expecting the kudo version to be validated as well.